config/routes.rb

```diff
@@ -1,4 +1,13 @@
 Rails.application.routes.draw do
+  root 'home#index'
+  # devise_for :users, :controllers => {:registrations => "my_devise/registrations"}
+  devise_for :users, controllers: { sessions: 'devise/sessions' }
+
+  
+  resources :languages do
+    delete 'remove'
+    post 'add'
```

Comment:
The languages controller doesn't contain an `add` method. Please consider deleting this line.